packages/forma-36-react-components/src/components/TextField/TextField.stories.tsx

```diff
@@ -59,49 +82,93 @@ WithTextLink.args = {
 };
 
 export const Overview = (args: TextFieldProps) => (
-  <>
-    <Flex marginBottom="spacingS">
-      <SectionHeading element="h3">
-        Textarea field default with countCharacters
-      </SectionHeading>
-    </Flex>
-    <Flex marginBottom="spacingS">
-      <TextField countCharacters {...args} />
-    </Flex>
-    <Flex marginBottom="spacingS">
-      <SectionHeading element="h3">
-        Textarea field default with value
-      </SectionHeading>
-    </Flex>
-    <Flex marginBottom="spacingS">
-      <TextField value="Example value" {...args} />
-    </Flex>
-    <Flex marginBottom="spacingS">
-      <SectionHeading element="h3">
-        Textarea field default with help text
-      </SectionHeading>
+  <Flex style={{ minWidth: '800px' }} flexDirection="column">
```

Comment:
Why is the `min-width` needed?

Reply:
I just thought it looks better

I did this grid with two columns, one for just input and the other for textarea
But IMO the inputs were really small, so I increased the width of this example's container

But I can change, I'm not really attached to this idea hahaha

Reply:
Agree, looks so much better 👏